README.md

```diff
@@ -53,9 +53,9 @@ file.
 ### Persist/Timestamp/Timeout
 
 The persist feature is disabled by default and can be enabled with the configure
-flag `--with-timestamp`.
+flag: `--with-timestamp`.
```

Comment:
"with the `--with-timestamp` configure flag" ?

Reply:
Incorporated.